docs/releasehistory.rst

```diff
@@ -1,12 +1,23 @@
 Release History
 ***************
 
-0.21.2 - Bugfix release
+0.21.3 - Bugfix release
 =======================
 
 Bugfixes
 --------
-- Fixed bug with samplers not broadcasting mixed replicas to MPI context (`#449 <https://github.com/choderalab/openmmtools/pull/449>`_).
+- Fixed long-standing bug with replica mixing using multiple GPUs  (`#449 <https://github.com/choderalab/openmmtools/pull/449>`_) & (`#562  <https://github.com/choderalab/openmmtools/pull/562>`_).
+- Add velocities to checkpoint file (`#555 <https://github.com/choderalab/openmmtools/pull/555>`_).
+- Fix documentation building (`#554 <https://github.com/choderalab/openmmtools/pull/554>`_).
+- Fix failing windows CI (`#573 <https://github.com/choderalab/openmmtools/pull/573>`_)
+
+Enhancements
+------------
+- Real time MBA analysis and timing information output in yaml format (`#565 <https://github.com/choderalab/openmmtools/pull/565>`_), (`#561 <https://github.com/choderalab/openmmtools/pull/561>`_) & (`#572 <https://github.com/choderalab/openmmtools/pull/572>`_).
```

Comment:
What is "MBA"? Do you mean "MBAR"?

Reply:
Let's make this an explicit statement:
"Real time MBAR analysis and timing information is now produced in yaml format at user-specified intervals"